src/test/java/guitests/MenuControlTest.java

```diff
@@ -117,6 +117,9 @@ public void menuControlTest() {
 
         // Testing First Panel selected
         assertEquals(0, (int) panelControl.getCurrentlySelectedPanel().get());
+        press(KeyCode.CONTROL).press(KeyCode.DIGIT1).release(KeyCode.DIGIT1).release(KeyCode.CONTROL);
+        assertEquals(0, (int) panelControl.getCurrentlySelectedPanel().get());
+        // TODO (currently not possible) test that panelsScrollPane.getHValue() is 0
```

Comment:
Okay, I see. Agreed that the test is good to have. The ScrollPane is passed to MenuControl, but it doesn't originate there -- the UI actually manages it. If you can get a UI reference, you could add some way of accessing it.

(The MenuControl probably doesn't need to have a direct reference to the ScrollPane... it should communicate with it either via the UI reference or event.)

On a side note, this test might be named wrongly. It should be PanelControlTest?